src/modules/flight_mode_manager/tasks/Orbit/flight_task_orbit_params.c

```diff
@@ -39,6 +39,19 @@
  * @max 10000.0
  * @increment 0.5
  * @decimal 1
- * @group FlightTaskOrbit
+ * @group Flight Task Orbit
  */
 PARAM_DEFINE_FLOAT(MC_ORBIT_RAD_MAX, 1000.0f);
+
+/**
+ * Yaw behaviour during orbit flight.
```

Comment:
Perhaps include "default" to make it clear this can be overridden.

```suggestion
 * Default yaw during orbit.
```